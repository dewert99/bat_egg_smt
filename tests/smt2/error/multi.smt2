(declare-fun f Bool)
(assert f)