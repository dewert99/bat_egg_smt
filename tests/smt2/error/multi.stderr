unexpected token when parsing command at 1:16
(declare-fun f Bool)
               ^

unknown identifier `f` at 2:9
(assert f)
        ^

